addons/l10n_ar_withholding/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    l10n_ar_tax_ids = fields.One2many(
+        'l10n_ar.partner.tax',
```

Comment:
```suggestion
        'account.tax',
```

Reply:
we keep as we have now 'l10n_ar.partner.tax'